Kernel/TTY/SlavePTY.cpp

```diff
@@ -40,11 +40,11 @@ void SlavePTY::echo(u8 ch)
 {
     if (should_echo_input()) {
         auto buffer = UserOrKernelBuffer::for_kernel_buffer(&ch);
-        m_master->on_slave_write(buffer, 1);
+        [[maybe_unused]] auto result = m_master->on_slave_write(buffer, 1);
```

Comment:
We can also just use `(void)` in places like this, to avoid looking huge and silly :^)

Reply:
Aye, will use that next time. :)